engine/apps/api/serializers/alert_receive_channel.py

```diff
@@ -533,19 +262,18 @@ def to_internal_value(self, data):
             except SkipField:
                 pass
             else:
-                # Line because of which method is overriden
-                if validated_value is None and isinstance(field, WritableSerializerMethodField):
-                    set_value(ret, [field.field_name], validated_value)
-                else:
-                    set_value(ret, field.source_attrs, validated_value)
+                set_value(ret, field.source_attrs, validated_value)
```

Comment:
It seems it's ok to remove everything from 240 to 265 line, since it was overriden only for WritableSerializerMethodField